starter-code/css/main.css

```diff
@@ -1,8 +1,197 @@
 /*
 Colors:
 
-Text: 1A1A1A
+Text: #1A1A1A
 Green: #00B172
 White: #FFF
 
 */
+body {
+    /*border: 2px solid black; */
+    box-sizing: border-box;
+    color: #1A1A1A;
+    font-family: "proxima_nova_ltthin";
+    margin: 0;
+    padding: 0;
+}
+
+.bottom {
+    background-color: #00B172;
+    margin: 2vw;
+    padding: 4vw 0;
+}
+
+.bottom .center {
+    background-image: url("../images/spotify-icon-white.png");
+    background-repeat: no-repeat;
+    background-size: contain;
+    /*border: 2px solid black;*/
+    display: inline-block;
+    height: 10vw;
+    width: 10vw;
+}
+
+.bottom .left {
+    color: #FFF;
+    /*border: 2px solid black;*/
+    display: inline-block;
+    margin-left: 5vw;
+    width: 40vw;
+}
+
+.bottom .left > div {
+    width: 20vw;
+}
+
+.bottom .left h1 {
+    border-bottom: 3px solid #FFF;
+    color: #FFF;
+    display: inline-block;
+    font-family: 'proxima_nova_ltsemibold';
+    font-size: 2em;
+    width: auto;
+}
+
+.bottom .left h2 {
+    font-family: 'proxima_nova_ltsemibold';
+}
+
+.bottom .left p {
+    font-size: 1.2em;
+}
+
+.bottom .right {
+    background-image: url("../images/spotify-app.jpg");
+    background-repeat: no-repeat;
+    background-size: contain;
+    /*border: 2px solid red;*/
+    display: inline-block;
+    height: 60vh;
+    margin: 2vw 2vw;
+    width: 30vw;
+}
+
+.icon {
+    height: 10vw;
+    width: 10vw;
+
+}
+
+.logo {
+    /*border: 2px solid green;*/
+    background-image: url("../images/spotify-logo.png");
+    background-repeat: no-repeat;
+    background-size: contain;
+    /*display: block;*/
+    position: fixed;
+    height: 70px;
+    top: 2vh;
+    left: 2vw;
+    width: 200px;
+}
+
+.logoAndMenu {
+    background-color: #FFF;
+    /*border: 2px solid blue;*/
```

Comment:
Ángel, ojo con el naming de las clases! Sé que puede parecer un lío.

En **html** usamos: kebab-case. Ejemplo: logo-and-menu

En **Javascript** usamos camelCase: logoAndMenu

Reply:
ok